gui/widgets/matRad_WorkflowWidget.m

```diff
@@ -403,11 +433,11 @@ function btnOptimize_Callback(this, hObject, eventdata)
                 
                 pln = evalin('base','pln');
                 ct  = evalin('base','ct');
-                
+                dij = evalin('base','dij');
                 % optimize
-                [resultGUIcurrentRun,usedOptimizer] = matRad_fluenceOptimization(evalin('base','dij'),evalin('base','cst'),pln);
+                [resultGUIcurrentRun,usedOptimizer] = matRad_fluenceOptimization(dij,evalin('base','cst'),pln);
```

Comment:
why did you take out all structures with the evalin but leave the cst in here?

Reply:
The cst is the only single use variable here, the others have been used at least one other time

Reply:
for readability we "take out" everything ?